docs/quick-start.md

```diff
@@ -100,9 +99,9 @@ numalogic-simple-pipeline-in-0-tmd0v               1/1     Running   0
 
 Once the pipeline has been created, the data can be sent to the pipeline by port-forwarding the input vertex.
 
-1. Port-forward to the http-source vertex
+1. Port-forward to the http-source vertex. From the above pod output, this would be:
    ```shell
-   kubectl port-forward simple-numalogic-pipeline-in-0-xxxxx 8443
+   kubectl port-forward numalogic-simple-pipeline-in-0-tmd0v 8443
```

Comment:
Do not give `tmd0v`, either indicate it needs to be changed clearly or something ppl know it needs to be changed.